### DISCUSSION TITLE
Is it compatible with Filament Version 3 (Tenant id)?

### DISCUSSION THREAD

**Discussion by Hegabovic** [saade/filament-fullcalendar#147] [Q&A]
it always give an error because it need the tenant id when creating a new event ? is there any way to pass the tenant_id in the query as the tenant id is always available using ``` Filament::getTenant()->id```

**Comment by almasrient** [comment]
I did use fullcalendar inside filamentphp v3, but

1. not yet applying tenant id
2. inserted event record was not populated on the calendar view

Maybe we can share our code to check any miss?

**Reply by Hegabovic** [reply]
I did it but the only problem i'm facing now is drage and drop givving me 
```id is null on array idk what is this error```
